cypress/e2e/FooterNavigation.cy.js

```diff
@@ -0,0 +1,14 @@
+import Home from '../src/pages/Home';
+import Footer from '../src/components/Footer';
+
+const homePage = new Home();
+const footer = new Footer();
```

Comment:
Why do you use the new Footer()? Insted of chining methods with return new Footer();